tests/test_job.py

```diff
@@ -61,7 +61,7 @@ def test_job_submit_output(hq_env: HqEnv):
     hq_env.start_server()
 
     output = hq_env.command(["submit", "--", "bash", "-c", "echo 'hello'"])
-    assert output.strip() == "Job submitted successfully, job ID: 1"
+    assert "Job submitted successfully, job ID: 1" in output.strip()
```

Comment:
Don't forget to revert this once you remove the printing of resources after submit.